notebooks/ch14/bias_modeling.md

```diff
@@ -215,14 +254,20 @@ plt.title('Irreducible error');
+
+As we have seen, however, achieving a low training error does not necessarily mean that our model will have a low test error as well. It is easy to obtain a model with extremely low bias and therefore low training error, by fitting a curve that passes through every training observation. However, this model will have high variance, which generally leads to high test error. Conversely, you can obtain a model with very low variance by fitting a horizontal line to the data, but this leads to high bias. Fundamentally, this occurs because training error reflects the bias of our model but not the variance; the test error reflects both. In order to minimize test error, our model needs to simultaneously achieve low bias and low variance. To account for this, we need a way to simulate test error without using the test set. This is generally done using cross validation.
+
+Cross-validation provides a more accurate method of estimating our model error using a single observed dataset by separating data used for training from the data used for model selection and final accuracy. Intuitively, the validation or test error checks the model's performance on a previously unseen dataset and thus allows us to estimate both bias and variance. Cross validation also allows us to incoporate the fact that the noise in the training set only affects the noise term in the risk, whearas the noise in the training set only affects bias and model variance. To choose the final model to use, we select the one that has the lowest validation error.
+
+The simplest form of cross-validation is the **train-validation split**. However, this method is prone to high variance, because the evaluation of the error may depend heavily on which points end up in the training and validation sets. To tackle this problem, we can run the train-validation split multiple times on the same dataset, using  **k-fold cross-validation**. The biggest advantage of this method is that every data point is used for validation exactly once, and for training *k-1* times. Typically, a *k* between 5 to 10 is used, but *k* remains an unfixed parameter. When *k* is small, the error estimate has a lower variance (many validation points), but has a higher bias (fewer training points). Vice versa, with large *k*, the error estimate will have lower bias, but have higher variance. K-fold cross-validation outputs a prediction error that more accurately reflects the bias and variance of a model. The model bias is the validation error. If this value is low, your model predicts values with low error on average. The model variance can be estimated by taking the standard error of the errors in each of the *k* trials. If this value is high, your model's performance varies a lot with the dataset used for training. 
```

Comment:
No comma needed in 
1) "this method is prone to high variance, because the evaluation of the error may depend" 
2) "dataset, using **k-fold cross-validation**"
3) "validation exactly once, and for training *k-1* times"
4) "(many validation points), but has a higher bias" [Perhaps take out "has" here as well.]
5) "lower bias, but have higher variance" [Same as before, perhaps take out "have"]
6) "Vice versa, with large *k*, the error estimate" [I think one of these two commas should be taken out.]

Combine these two sentences:
"K-fold cross-validation outputs a prediction error that more accurately reflects the bias and variance of a model. The model bias is the validation error."
(maybe with "such that")

Maybe use "our" instead of "your" in the last couple of sentences.

Reply:
4.  leaving has for parallelism 
I think combining the sentences would be warranted if I had something "K-fold cross-validation outputs a prediction error that more accurately reflects the bias and variance of a model such that the model bias is the validation error and the model variance is the standard error of the individual errors..." But the second statement isn't true, and I feel weird just leaving it at "K-fold cross-validation outputs a prediction error that more accurately reflects the bias and variance of a model such that the model bias is the validation error"

Reply:
I think for the latter just combining with ", and" is fine too; right now it seems a bit too choppy with the short sentence there.

Reply:
See updated comment. I see what you mean after reading it a a few more times.

Reply:
in particular sounds good